plugins/catalog-backend/src/service/DefaultEntitiesCatalog.ts

```diff
@@ -274,28 +271,6 @@ export class DefaultEntitiesCatalog implements EntitiesCatalog {
       entities = entities.map(e => request.fields!(e));
     }
 
-    // TODO(freben): This is added as a compatibility guarantee, until we can be
```

Comment:
@freben help me out here. This _should_ be okay to remove in a `patch` release right given that it's old enough to not have any end user impact, right? 😅

Reply:
After thinking about it more, I am going to bump this change to a `minor` release because it removes code.